<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 200), stop:1 rgba(240, 240, 240, 199));</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QStackedWidget" name="MenuPrincipal">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>1011</width>
         <height>721</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="ConsultarDatos_Button">
          <property name="text">
           <string>Consultar Datos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="TecnologiasSolicitadas_Button">
          <property name="text">
           <string>Consultar Tecnologías</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ConsultarHistorial_Button">
          <property name="text">
           <string>Consultar Soft Skills</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>311</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>src/LogoLinkJob.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="MenuPrincipal_Button">
          <property name="text">
           <string>Inicio</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>1011</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="GenerarDatos_Button">
            <property name="text">
             <string>Generar Nuevos Datos</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Visualizar fecha pasada</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Fecha_Button"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Escoger Rol de Ingeniero de Software</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Rol_Button"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Escoger Filtro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Filtro_Button"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="Grafica">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>220</y>
         <width>1001</width>
         <height>581</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="verticalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>321</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="MenuPrincipal_Button_2">
            <property name="text">
             <string>Inicio</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Selecciona una fecha</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Fecha_Button_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>431</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>Selecciona una fecha para consultar las tecnologías más solicitadas de esa fecha</string>
       </property>
      </widget>
      <widget class="QLabel" name="TecnologiasSolicitadasGrafica">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>1001</width>
         <height>581</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QWidget" name="verticalLayoutWidget_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="MenuPrincipal_Button_5">
          <property name="text">
           <string>Inicio</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Selecciona una Fecha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="Fecha_Button_5"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>471</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string>Selecciona una fecha para consultar la cantidad de softskills solicitadas de esa fecha</string>
       </property>
      </widget>
      <widget class="QLabel" name="ConsultarHistorialGrafica">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>1001</width>
         <height>581</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
